### PR TITLE
Add set of metrics to register model endpoint

### PR DESCRIPTION
**Pull request by Miro001**

### CODE REVIEW

File: proto/asgt/v2/dataset_service.proto [hunk 1]
```diff
@@ -95,88 +95,88 @@ service DatasetService {
 
 message GetDatasetRequest {
   string dataset_name = 1 [(validate.rules).string = {
-    pattern: "^[A-Za-z0-9.][A-Za-z0-9_.>-]*$",
+    pattern: "^[A-Za-z0-9.][A-Za-z0-9_.>-]*$"
```
Comment (michalderdak) [inline proto/asgt/v2/dataset_service.proto:98]: are the commas not needed?

Comment (michalderdak) [reply proto/asgt/v2/dataset_service.proto:98]: seems like you have a linter cleaning up things... we should align on that somehow

File: proto/asgt/type/target_metrics.proto [hunk 2]
```diff
@@ -17,4 +17,7 @@ message TargetMetrics {
 
   string target = 1;
   repeated Metric metrics = 2;
+  repeated float mcc = 3;
```
Comment (michalderdak) [inline proto/asgt/type/target_metrics.proto:20]: maybe a comment above this as to what mcc is?

**Review by michalderdak — APPROVED** [review]

File: proto/asgt/v2/dataset_service.proto (hunk 1 above)
Comment (Miro001) [reply proto/asgt/v2/dataset_service.proto:98]: latest version of buf i guess?

**Review by anton-cornak — APPROVED** [review]

**Pull request merged**